<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clue::ConfigureGame</class>
 <widget class="QDialog" name="Clue::ConfigureGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>600</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Your Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="p0Name"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string># of Players</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="numPlayers">
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="maximum">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>P1 Name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="p1Name"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>P2 Name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="p2Name"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>P3 Name</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="p3Name"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>P4 Name</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="p4Name"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>P5 Name</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="p5Name"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Who's First?</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QComboBox" name="whosFirst"/>
    </item>
    <item row="9" column="1">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Your Card 0</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QComboBox" name="card0"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Your Card 1</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QComboBox" name="card1"/>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Your Card 2</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QComboBox" name="card2"/>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Your Card 3</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QComboBox" name="card3"/>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Your Card 4</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QComboBox" name="card4"/>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Your Card 5</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QComboBox" name="card5"/>
    </item>
    <item row="7" column="1">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string># of Cards</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QSpinBox" name="numCardsInHand">
      <property name="minimum">
       <number>3</number>
      </property>
      <property name="maximum">
       <number>6</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
